### PR TITLE
Feature/refresh dashboard

### PR DESCRIPTION
**Pull request by jonasem**
https://nordicsemi.atlassian.net/jira/software/c/projects/NCD/boards/942?selectedIssue=NCD-254

### CODE REVIEW

File: src/components/SidePanel/TraceOptions.tsx [hunk 1]
```diff
@@ -31,27 +34,46 @@ export default () => {
...
+
+const RefreshOnStart = () => {
+    const dispatch = useDispatch();
+    const isTracing = useSelector(getIsTracing);
+    const refreshOnStart = useSelector(getRefreshOnStart);
+    const selectedFormats = useSelector(getTraceFormats);
+    const liveEnabled = selectedFormats.includes('live');
+    const title = liveEnabled
+        ? 'Cannot be enabled when Wireshark is enabled.'
+        : 'Refresh dashboard 5 seconds after starting the trace';
```
Comment (aadnekar) [inline src/components/SidePanel/TraceOptions.tsx:68]: The specifications says to have `Cannot be enabled when Wireshark is enabled.`, even when Wireshark is not enabled. I think it's for consistency, but I do also like that detailed description. Could this be merged as one text? Something like this:

```typescript
const title = 'Cannot be enabled when Wireshark is enabled.\nRefresh dashboard 5 seconds after starting the trace';
```

File: src/features/tracing/nrfml.ts [hunk 2]
```diff
@@ -238,6 +240,11 @@ export const startTrace =
             }
         }
 
+        if (getRefreshOnStart(state) && !formats.includes('live')) {
+            logger.info(`Refreshing values in 5 seconds`);
```
Comment (aadnekar) [inline src/features/tracing/nrfml.ts:244]: I suggest we change values to dashboard, in order to promote consistency.

```suggestion
            logger.info(`Refreshing dashboard in 5 seconds`);
```

File: src/utils/store.ts [hunk 3]
```diff
@@ -94,3 +95,8 @@ export const [restoreResetDevice, storeResetDevice] = fromStore(
     'resetDevice',
     false
 );
+
+export const [restoreRefreshOnStart, storeRefreshOnStart] = fromStore(
+    'refreshOnStart',
+    false
```
Comment (aadnekar) [inline src/utils/store.ts:101]: In order to support the Quick Start project, we want it to refresh on start by default.

```suggestion
    true
```

**Review by aadnekar — COMMENTED** [review]
The Refresh of start should be enabled by default, and I've added two very minor suggestions related to text.

Could you also verify that wireshark is disabled by default, so that we do not end up having an 'illegal' state by default? 🦹
 
 Thank you! 🤗

**Comment by jonasem** [comment]
> The Refresh of start should be enabled by default, and I've added two very minor suggestions related to text.
> 
> Could you also verify that wireshark is disabled by default, so that we do not end up having an 'illegal' state by default? 🦹
> 
> Thank you! 🤗

Verified that it is disabled by default (by reseting store). Thanks for the good feedback! 👍

**Review by aadnekar — APPROVED** [review]
🚀

**Pull request merged**